Shownotes/Episode1-Transcript.vtt

```diff
@@ -0,0 +1,710 @@
+
+00:28:47.238 --> 00:29:26.998
+<v Chris Lattner>Yeah, absolutely. The reason it is conflicting to me is that from the beginning of the project, my goal was to build a full-stack system. It was to look at all the existing systems out there, see what's good or bad about each of them, and then cherry-pick the best ideas from systems wherever they come from. And the goal was really to build something that you could write firmware in or that you could do scripting in, that could write mobile apps or server apps or low-level systems code, and have it be great at all of those, not just some terrible compromise. And so that positioning was absolutely the right thing to do.
+
+00:29:27.718 --> 00:29:35.298
+<v Chris Lattner>But hopefully, Swift will grow over time in kinds of what it is able to do.
+
+00:29:35.538 --> 00:29:44.938
+<v Garric Nahapetian>I was sitting at home watching. I wasn't at dub-dub. I was still just an outside observer. And when it was announced, I had no idea what it meant.
+
+00:29:45.528 --> 00:29:54.628
+<v Garric Nahapetian>All I knew is that people were excited. And people were also unsure about what it was like, I guess. Or they were questioning.
+
+00:29:54.848 --> 00:29:57.778
+<v John Sundell>It was excitement mixed with anxiety, right?
+
+00:29:57.838 --> 00:30:32.178
+<v Garric Nahapetian>And I just remember not knowing what it meant. All I really knew is that it was a new and shiny. And I wanted to just look into it and learn about it. And so I remember trying to listen to every podcast I could get my hands on, or every article, to try to understand what it meant. Ultimately, what it meant for me, and I only discovered this maybe six months later, was that it was going to be more accessible for me. It was going to enable me to give myself another shot at making an iPhone app.
+
+00:30:32.658 --> 00:31:09.181
+<v Chris Lattner>That's a perfect segue into the last group I want to talk about. Because, you look at what Swift is and sure, it's a compiler, it's a language spec, it's a set of APIs, it's and IDE. But the thing that makes it real and the thing that reached so many people is the work done by the Developer Publication Group. These are the tech writers at Apple that wrote things such as the Swift Programming Language book. And the success and the rapid adoption of Swift is in huge part directly related to
+
+00:31:10.071 --> 00:31:18.041
+<v Chris Lattner>such high quality good documentation and a book being available on day one. And it's continued to be maintained today, and it's amazing.
+
+00:31:18.571 --> 00:31:34.451
+<v Chris Lattner>And part of that was that we pulled in the tech writers right into the design meetings. Folks like Tim Isted and Brian Lanyon[?], Alex Martini spent a tremendous amount of time in weekly meetings where we were arguing about, "do we use
```

Comment:
@lattner I'm not quite sure I got this name right: "Brian Lanyon". Can you confirm?